substrate/frame/identity/src/lib.rs

```diff
@@ -947,7 +957,9 @@ pub mod pallet {
 			let sub = T::Lookup::lookup(sub)?;
 			ensure!(IdentityOf::<T>::contains_key(&sender), Error::<T>::NoIdentity);
 			ensure!(SuperOf::<T>::get(&sub).map_or(false, |x| x.0 == sender), Error::<T>::NotOwned);
-			SuperOf::<T>::insert(&sub, (sender, data));
+			SuperOf::<T>::insert(&sub, (sender.clone(), data));
```

Comment:
```suggestion
			SuperOf::<T>::insert(&sub, (&sender, data));
```

Reply:
So there is some magic happening with storage operations where you can pass a reference to the data, and since the value need to be encoded anyway into a vec, we do a clone on that side.

So you can just pass references, and avoid clones in this way.